docs/content/main/troubleshooting.md

```diff
@@ -6,16 +6,16 @@ weight = 4
 
 ## Troubleshooting
 
-Here you will find help if something doesn't work, if the issue isn't listed here please [open an issue on the github repo.](https://github.com/elkowar/eww/issues)
+Here you will find help if something doesn't work, if the issue isn't listed here please [open an issue on the GitHub repo.](https://github.com/elkowar/eww/issues)
 
 ### My scss isn't being loaded!
 
 1. You have not created a scss file
 2. The scss file isn't called correctly. (it should be called `eww.scss` in the `$HOME/.config/eww` folder)
 3. The scss file isn't placed in the correct location (check above)
-4. You have used two (or more) semi-colons (;;) insted of one (;) at the end of a line.
+4. You have used two (or more) semi-colons (;;) instead of one (;) at the end of a line.
```

Comment:
```suggestion
```

Tbh this is just wrong, double semicolon should never make a difference. I probably overlooked this in the original PR that added the docs.